acl_test.go

```diff
@@ -108,3 +110,94 @@ func TestAuthorized(t *testing.T) {
 	assert.True(t, isAuth)
 	assert.Nil(t, err)
 }
+
+func TestDefaultACEHasPermissionBoundaries(t *testing.T) {
+	ace := NewACE("00000000-0000-0000-0000-000000000000", Permission(1))
+	isAuth, err := ace.HasPermission(Permission(-1))
+	assert.False(t, isAuth, "should not have permissions")
+	assert.Nil(t, err, "should be no error")
```

Comment:
Testify has an assert.NoError() (and assert.Error() which makes the assertion of errors a little easier to read.